pom.xml

```diff
@@ -49,6 +49,7 @@
 		<module>psl-sampler</module>
 		<module>psl-weightlearning</module>
 		<module>psl-mosek</module>
+		<module>psl-clj</module>
```

Comment:
Can we name it "psl-clojure" instead of "psl-clj"?
I have two reasons:
 1. "clj" is pretty close to "cli".
 2. We already have "psl-groovy" which I guess uses both the extension and language name, but I always interpreted as the language name.

Reply:
Done